tests/unit/amo/components/TestRatingManager.js

```diff
@@ -537,6 +582,8 @@ describe(__filename, () => {
   });
 
   describe('mapStateToProps', () => {
```

Comment:
Since you ported to `shallowUntilTarget`, these tests can all either be deleted or switched over to working against `render()`. Rejoice! 🌴  

One way to approach it is to start breaking things and see what tests are already covering their logic. You may need to add a few tests where the logic is not fully covered.

```diff
diff --git a/src/amo/components/RatingManager/index.js b/src/amo/components/RatingManager/index.js
index 10087dda6..662a995c0 100644
--- a/src/amo/components/RatingManager/index.js
+++ b/src/amo/components/RatingManager/index.js
@@ -270,7 +270,7 @@ export const mapStateToProps = (state: AppState, ownProps: Props) => {
   }
 
   return {
-    apiState: state.api,
+    //apiState: state.api,
     userReview,
     userId,
   };
diff --git a/tests/unit/amo/components/TestRatingManager.js b/tests/unit/amo/components/TestRatingManager.js
index e771e4bf2..bfb8fda2b 100644
--- a/tests/unit/amo/components/TestRatingManager.js
+++ b/tests/unit/amo/components/TestRatingManager.js
@@ -581,7 +581,7 @@ describe(__filename, () => {
     });
   });
 
-  describe('mapStateToProps', () => {
+  describe.skip('mapStateToProps', () => {
     let store;
 
     function getMappedProps({

```

Reply:
> You may need to add a few tests where the logic is not fully covered.

Looks like I did not have to write new test cases.